lib/src/revset.rs

```diff
@@ -928,6 +930,12 @@ static BUILTIN_FUNCTION_MAP: Lazy<HashMap<&'static str, RevsetFunction>> = Lazy:
             ))
         }
     });
+    map.insert("conflicts", |name, arguments_pair, _state| {
+        expect_no_arguments(name, arguments_pair)?;
```

Comment:
Nit: the template keyword is singular `conflict`, but maybe the convention in revset is plural?

Reply:
Good question. It seems natural for e.g. `author(someone)` to be singular. I think the reason is that the needle is matched against the commit's single author; it's clearly not expected to resolve to a single commit. I think `tags()` and `merges()` feel more natural as plural, but I see that Mercurial uses singular for them (`tag()` and `merge()`). They seem to use singular for revsets functions that don't take any input set and plural for functions that do. I'm not sure what the rational for that is, but maybe we should switch all revset functions to be singular?

Reply:
Oh, maybe the logic is that if the plural refers to *revisions*, then it should be used. `ancestors()` is about multiple ancestor revisions, `branches()` and `tags()` are about multiple branches or tags (i.e. not revisions). By that logic, this new function should be `conflict()` as you suggested, and the existing `merges()` should remain plural even though it's different from Mercurial. We would then rename the existing `branches()`, `remote_branches()`, `tags()` and `git_refs()`. If we do that, we could still leave them as aliases for a while, or even forever. That might be helpful and harmless either way. I'm still not sure what I think, but I'll leave this PR open until we've decided.

Reply:
I've renamed `conflicts()` to `conflict()`. I'll leave renaming of `branches()` etc. for later.